frontend/src/v5/ui/routes/dashboard/projects/drawings/drawingDialogs/createDrawingDialog.component.tsx

```diff
@@ -16,33 +16,24 @@
  */
 
 import { formatMessage } from '@/v5/services/intl';
-import { FormTextField, FormSelect } from '@controls/inputs/formInputs.component';
-import { MenuItem } from '@mui/material';
 import { SubmitHandler } from 'react-hook-form';
 import { FormModal } from '@controls/formModal/formModal.component';
-import { DrawingsHooksSelectors, ProjectsHooksSelectors, TeamspacesHooksSelectors } from '@/v5/services/selectorsHooks';
+import { ProjectsHooksSelectors, TeamspacesHooksSelectors } from '@/v5/services/selectorsHooks';
 import { DrawingsActionsDispatchers } from '@/v5/services/actionsDispatchers';
-import { nameAlreadyExists, numberAlreadyExists } from '@/v5/validation/errors.helpers';
-import { UnhandledErrorInterceptor } from '@controls/errorMessage/unhandledErrorInterceptor/unhandledErrorInterceptor.component';
 import { IFormInput, useDrawingForm } from './drawingsDialogs.hooks';
+import { DrawingForm } from './drawingForm.component';
+import { DEFAULT_SETTINGS_CALIBRATION } from '../../calibration/calibration.helpers';
 
 export const CreateDrawingDialog = ({ open, onClickClose }) => {
 	const teamspace = TeamspacesHooksSelectors.selectCurrentTeamspace();
 	const project = ProjectsHooksSelectors.selectCurrentProject();
-	const types = DrawingsHooksSelectors.selectTypes();
 
-	const { onSubmitError, formData } = useDrawingForm();
-
-	const {
-		handleSubmit,
-		control,
-		formState,
-		formState: { errors },
-	} = formData;
+	const { onSubmitError, formData } = useDrawingForm({ calibration: DEFAULT_SETTINGS_CALIBRATION } as any);
+	const { handleSubmit, formState } = formData;
 
 	const onSubmit: SubmitHandler<IFormInput> = async (body) => {
 		try {
-			await new Promise<void>((accept, reject ) => DrawingsActionsDispatchers.createDrawing(teamspace, project, body as any, accept, reject));
+			await new Promise<void>((accept, reject) => DrawingsActionsDispatchers.createDrawing(teamspace, project, body as any, accept, reject));
```

Comment:
- when calibrating the last step doesnt seems to respect the actual min/max plane that I've set when creating the drawing, both planes stays at 0
<img width="1034" alt="image" src="https://github.com/user-attachments/assets/0edc7c05-4d8f-4a95-b781-2e59c7f7ae21">